components/script/dom/htmlelement.rs

```diff
@@ -560,11 +560,11 @@ static DATA_PREFIX: &str = "data-";
 static DATA_HYPHEN_SEPARATOR: char = '\x2d';
 
 fn is_ascii_uppercase(c: char) -> bool {
-    'A' <= c && c <= 'Z'
+    c.is_ascii_uppercase()
 }
 
 fn is_ascii_lowercase(c: char) -> bool {
```

Comment:
This function is now redundant, as it is not doing any actual work itself and simply forwards the call to the standard library method. So, we can just delete this function definition from this file and simply call `.is_ascii_lowercase()`  wherever the deleted function is used. Please refer to my previous comment for an example of what needs to be done at the call site. 

Similar refactoring needs to done for `is_ascii_uppercase` as well.